tests/unit/algorithms/action/utils/test_action_convert_public_data_to_cvat.py

```diff
@@ -0,0 +1,171 @@
+"""Unit Test for otx.algorithms.action.utils.convert_public_data_to_cvat."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import numpy as np
+from lxml import etree
```

Comment:
This seems like a security issue. Can you check it out?
![image](https://user-images.githubusercontent.com/38045080/218616548-3bdebe3d-b45c-4fa0-bc56-4f6c33419d7f.png)

Reply:
> This seems like a security issue. Can you check it out? ![image](https://user-images.githubusercontent.com/38045080/218616548-3bdebe3d-b45c-4fa0-bc56-4f6c33419d7f.png)

Thanks for the nice finding ! 
After some researches, I found that we are in the safe environment since we are using self-made XML files. 

This vulnerability comes from maliciously crafted XML.

Reference: https://docs.python.org/3/library/xml.html#xml-vulnerabilities

Reply:
At this time, removing lxml in original code is too hard. Therefore I remove lxml in the test codes first.